data/card_catalogue/unique-1.json

```diff
@@ -224,38 +277,50 @@
     "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "Summon a diembodied hand within 4 tiles to perform one simple action before vanishing.",
-      "flavour": "\"Magicians are a bunch of lazy bums\" - Brinley B.",
+      "rules": {
+        "active": "Evoke a disembodied hand to perform a simple action within 4 metres of you.",
+        "passive": ""
+      },
+      "flavour": "Give me five!",
       "image": "assets/core/card/illustration/Eidolimb.png"
     }
   },
   {
-    "title": "Tripwire Charm",
+    "title": "Espial Thread",
     "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "Choose two visible points. You will be alerted the next time an NPC walks between them.",
-      "flavour": "Mafia hideouts are often mysteriously empty when Rebel forces arrive to perform arrests.",
+      "rules": {
+        "active": "Choose two visible points. You will be alerted the next time an NPC walks between them.",
+        "passive": ""
+      },
+      "flavour": "Mob hideouts are often mysteriously empty when the Cops arrive to perform arrests",
       "image": "assets/core/card/illustration/Tripwire Charm.png"
     }
   },
   {
-    "title": "Large-Caliber Artillery Invocation",
+    "title": "Spectreblast",
     "upgrade": "",
     "tier": "green",
     "details": {
-      "rules": "A magical light takes the shape of a cannon and instantly fires a real shot in the direction of your choice.",
-      "flavour": "Cannonballs are designed to destroy ships and stone fortresses. The effect they have on the human body is unspeakable.",
-      "image": "assets/core/card/illustration/Large-Caliber Artillery Invocation.png"
+      "rules": {
+        "active": "Fire a cannonball at a target of your choice.",
+        "passive": ""
+      },
+      "flavour": "Cannonballs were designed to destroy ships and stone fortresses. The effect they have on the human body is unspeakable.",
+      "image": "assets/core/card/illustration/Spectreblast.png"
     }
   },
   {
     "title": "PBJT",
     "upgrade": "",
     "tier": "blue",
     "details": {
-      "rules": "A visible character of your choice receives a sandwich to the face.",
-      "flavour": "After taking a sandwich to the face from one of his men, Coronel Kudzo was forced to retire, effective immediately",
+      "rules": {
+        "active": "Apply a sandwich to the face of a visible character.",
```

Comment:
This is one of the few instances where I changed the wording as per https://github.com/elliottomlinson/rpcg/pull/111#issuecomment-892889072 to fit the 
> ✋ = *"Casting this card lets you..."* 

scheme, but honestly I just thought "apply to the face" was a funny thing to say about a sandwich. dc4de5f